Applications/editor/assets/ui/header.wns

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Why the empty file?

Reply:
Same as above. For now: To make sure that the `#includes` are getting hit.